packages/bot-web-ui/src/constants/contract.js

```diff
@@ -2,39 +2,6 @@ import { localize } from '@deriv/translations';
 import { getTotalProfit } from '@deriv/shared';
 import { getBuyPrice } from 'Utils/multiplier';
 
-export const getCardLabels = () => ({
```

Comment:
@jim-deriv I noticed that we have `getCardLabels` in the following files as well:

>  `packages/trader/src/Constants/contract.js`
>  `packages/reports/src/_common/contract.js` 

  
  So can we change remove above files as well?
  if so, then we can also change the import path for `getCardLabels` in the following files: 
  
  **for Trader workspace:** 
  

> `packages/trader/src/App/Containers/ProgressSliderStream/progress-slider-stream.jsx`
>   `packages/trader/src/App/Components/Elements/ContractDrawer/contract-drawer-card.jsx`
>   `packages/trader/src/App/Components/Elements/PositionsDrawer/positions-modal-card.jsx`
>   `packages/trader/src/App/Containers/ProgressSliderStream/progress-slider-stream.jsx`
>   `packages/trader/src/Modules/Trading/Components/Form/TradeParams/Multiplier/expiration.jsx`


**for Reports workspace:**

> `packages/reports/src/Components/indicative-cell.jsx` 
> `packages/reports/src/Constants/data-table-constants.js`
> `packages/reports/src/Containers/open-positions.jsx`
> `packages/reports/src/Containers/progress-slider-stream.jsx`

Reply:
Hello @iman-fs,

 packages/trader/src/Constants/contract.js is already gone.
 
 (packages/trader/src/Constants/contract.js) This one, I'll replace

But our goal here is not to remove all the duplicates at once, because they're many!
almost all packages have duplicates and I've requested @ali-hosseini-fs to let me remove the duplicates in bits, otherwise, if I remove too much at once and it results into a bug, it'll be very hard to trace, yet the main goal was to fix sonarcloud. I'll be removing the duplicates bit by bit, but as of now, when reviewing this PR, please just help me confirm that my current changes are not affecting the system negatively. There will be another PR to remove the rest of the duplicates e.g. `loaders-config.js` , `base-store.js`, among others which are in every package and requires careful attention

Removing duplicates in bits is reasonable because, they're harder to maintain, but don't negatively impact the system unless they surpass sonarcloud's limit

What do you think?

Reply:
I agree with you. 
specially for cases like stores. It's better to do it little by little as you said.